etcdutl/snapshot/v3_snapshot.go

```diff
@@ -174,6 +186,15 @@ func (s *v3Manager) Status(dbPath string) (ds Status, err error) {
 	return ds, nil
 }
 
+func bytesToRev(b []byte) (rev mvcc.Revision, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = fmt.Errorf("%s", r)
+		}
+	}()
+	return mvcc.BytesToRev(b), nil
```

Comment:
```suggestion
	return mvcc.BytesToRev(b), err
```

Reply:
nil is not wrong here. err is overriden in recover func.

Reply:
Yes, you are right.

Reply:
Still, the change proposed by @ahrtr would make the intention of the code more clearer.

Reply:
I've updated it.

Reply:
`err` is only used in defer.
We can use `retErr` to distinguish `err` from return.